### DISCUSSION TITLE
Design fire for rack storage without rack sprinkler

### DISCUSSION THREAD

**Discussion by quantumfds** [firemodels/fds#13603] [Q&A] [answered]
Hello,

I would like to design a fire in a huge rack storage room which is currently without in-rack sprinklers. There is only normal sprinkler system installed on the ceiling above the racks. The idea is to cap the heat release rate once the sprinkler has been activated and investigate further the smoke extraction.
Would it be plausible to model the rack and put the fire unter the first rack and see when the sprinkler will be activated? Or is there a better method to determine the heat release rate which has to be capped?
What would be the best to investigate spread of fire in the rack itself? My idea was to use the fast growth rate and cap it once the sprinkler has been activated. But this is not very conservative method, since the ceiling sprinkler would not be that effective in this case (that is why the in-rack sprinkler would be much better option).

Thank you.

**Comment by drjfloyd** [comment, marked as answer]
We can't tell you what is best for your specific use of FDS. That is a decision that you need to make and justify as the modeler in conjunction with any stakeholders whose input / concurrence is required.
